setup/install/storage/minio.md

```diff
@@ -29,6 +29,14 @@ reachable by Spinnaker. Record the following values:
 
 ## Editing your storage settings
 
+Given that Minio doesn't support versioning objects, we need to disable it
+in Spinnaker. Add the following line to `~/.hal/$DEPLOYMENT/profiles/front50-local.yml`
```

Comment:
Put a colon after `front50-local.yml`
and move the sentence about `$DEPLOYMENT` being `default` to _after_ the command.